app/routers/mail.py

```diff
@@ -0,0 +1,38 @@
+from fastapi import BackgroundTasks, APIRouter, Depends
+from pydantic import BaseModel, EmailStr
+from pydantic.errors import EmailError
+
+import config
+from internal.mail import send_fast_email_invitation
+
+router = APIRouter()
+
+CORRECT_EMAIL_ADDRESS_ERROR_MESSAGE = "Please enter valid email address"
```

Comment:
Should be renamed to 'incorrect' or 'invalid' rather than 'correct'

Reply:
I will corrent this. thank you